driver.go

```diff
@@ -127,6 +127,7 @@ func (d SnowflakeDriver) OpenWithConfig(ctx context.Context, config Config) (dri
 	sc.populateSessionParameters(authData.Parameters)
 	sc.ctx = context.WithValue(sc.ctx, SFSessionIDKey, authData.SessionID)
 	sc.startHeartBeat()
+	sc.internal = &internalClient{sr: sc.rest}
```

Comment:
on successful connection creation, stash the `internalClient` into the field `internal`